### PR TITLE
11.0 mig account banking sepa credit transfer

### PR DESCRIPTION
**Pull request by etobella**
Migration to 11.0.
Improved tests
Depends on:
 - [x] account_payment_mode #399
 - [x] account_payment_partner #400
 - [x] account_payment_order #401
 - [x] account_payment_pain_base #405

### CODE REVIEW

**Review by rgarnau — APPROVED** [review]
Code and functional review :+1:

**Review by HviorForgeFlow — APPROVED** [review]
Functional Review

**Comment by angelmoya** [comment]
Functional tested OK :+1:

File: account_banking_sepa_credit_transfer/README.rst [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+Module to export payment orders in SEPA XML file format.
+
+SEPA PAIN (PAyment INitiation) is the new european standard for
+Customer-to-Bank payment instructions. This module implements SEPA Credit
+Transfer (SCT), more specifically PAIN versions 001.001.02, 001.001.03,
+001.001.04 and 001.001.05. It is part of the ISO 20022 standard, available on
+http://www.iso20022.org.
+
+The Implementation Guidelines for SEPA Credit Transfer published by the
+European Payments Council (http://http://www.europeanpaymentscouncil.eu) use
```
Comment (AaronHForgeFlow) [inline account_banking_sepa_credit_transfer/README.rst]: The link in this line is not working. Maybe this? https://www.europeanpaymentscouncil.eu/

**Comment by etobella** [comment]
It has been rebased in order to remove dependencies and all is green

**Pull request merged**